.github/workflows/database-restore.yml

```
@@ -34,12 +35,37 @@ jobs:
          INSTALL_CONDUIT: true
 
       - name: Backup Teacher Training API Prod Database
-        run: cf conduit teacher-training-api-postgres-prod -- pg_dump --encoding utf8 --clean --no-owner --if-exists -f prod_backup.sql
-      
+        run: |
+          now=$(date +"%S_%M_%H_%m_%d_%Y")
+          prod_sql_file=prod_backup-$now.sql
```

```suggestion
          prod_sql_file=prod_backup-$now.sql
          echo "PROD_BACKUP=$prod_sql_file" >> $GITHUB_ENV
```
and use `${{ env.PROD_BACKUP }}` in the below steps